pkg/scheduler/framework/session_plugins.go

```diff
@@ -293,3 +297,26 @@ func (ssn *Session) PredicateFn(task *api.TaskInfo, node *api.NodeInfo) error {
 	}
 	return nil
 }
+
+func (ssn *Session) PriorityFn(task *api.TaskInfo, node *api.NodeInfo) (int, error) {
+	priorityScore := make(map[string]int)
```

Comment:
why use map?

Reply:
Once weight is supported, if we are giving different weight to different plugins, it will be easy to calculate score with map.  I will change it to int and return the total sum as score

Reply:
when support weight, we also only return the summary.